flows/actions/call_llm.go

```diff
@@ -55,11 +61,9 @@ func NewCallLLM(uuid flows.ActionUUID, llm *assets.LLMReference, instructions, i
 func (a *CallLLMAction) Execute(ctx context.Context, run flows.Run, step flows.Step, logModifier flows.ModifierCallback, logEvent flows.EventCallback) error {
 	resp := a.call(ctx, run, logEvent)
 	if resp != nil {
-		run.Locals().Set("_llm_status", "success")
-		run.Locals().Set("_llm_output", resp.Output)
+		run.Locals().Set(LLMOutputLocal, resp.Output)
 	} else {
-		run.Locals().Set("_llm_status", "failure")
-		run.Locals().Set("_llm_output", "")
+		run.Locals().Set(LLMOutputLocal, LLMOutputFailed)
```

Comment:
I forgot that router rules have one operand... so need a single value that describes successful output as well as failure